README.md

```diff
@@ -122,7 +122,7 @@ next rails s        # Start server using Gemfile.next
 
 ## Installation
 
-Add this line to your application's Gemfile:
+Add this line to your application's Gemfile (if you add this gem to a group, make sure it is the test env group):
```

Comment:
Hey @rishijain what do you think about this? otherwise, it looks good to me :+1: 

```suggestion
Add this line to your application's Gemfile

> NOTE: If you add this gem to a group, make sure it is the test env group
```

Reply:
thanks @JuanVqz ! updated.